src/index.js

```diff
@@ -1,73 +1,3 @@
-import Raw from './raw';
-import Client from './client';
+const Knex = require('./knex');
 
-import makeKnex from './util/make-knex';
-import parseConnection from './util/parse-connection';
-
-import { assign } from 'lodash';
-
-// The client names we'll allow in the `{name: lib}` pairing.
-const aliases = {
-  pg: 'postgres',
-  postgresql: 'postgres',
-  sqlite: 'sqlite3',
-};
-
-export default function Knex(config) {
-  if (typeof config === 'string') {
-    return new Knex(assign(parseConnection(config), arguments[2]));
-  }
-  let Dialect;
-  if (arguments.length === 0 || (!config.client && !config.dialect)) {
-    Dialect = Client;
-  } else if (
-    typeof config.client === 'function' &&
-    config.client.prototype instanceof Client
-  ) {
-    Dialect = config.client;
-  } else {
-    const clientName = config.client || config.dialect;
-    Dialect = require(`./dialects/${aliases[clientName] ||
-      clientName}/index.js`);
-  }
-  if (typeof config.connection === 'string') {
-    config = assign({}, config, {
-      connection: parseConnection(config.connection).connection,
-    });
-  }
-  return makeKnex(new Dialect(config));
-}
-
-// Expose Client on the main Knex namespace.
-Knex.Client = Client;
-
-/* eslint no-console:0 */
-
-Object.defineProperties(Knex, {
-  VERSION: {
-    get() {
-      console.warn(
-        'Knex.VERSION is deprecated, you can get the module version' +
-          "by running require('knex/package').version"
-      );
-      return '0.12.6';
```

Comment:
Was this just dropped in this PR? It's hard to follow all the changes after renaming... I need to diff the files locally. If this was dropped, it must be notified in changelog as breaking change.

Reply:
I've added comments in the PR for all the changes after renaming; this one was done after @ricardograca comment. If you are ok with this change, I'll add it to changelog.

Reply:
Yep, I'm fine with it. I'll check out diff locally later on.